src/components/molecules/Visualizer/Engine/Cesium/useEngineRef.ts

```diff
@@ -87,6 +87,13 @@ export default function useEngineRef(
           options,
         );
       },
+      lookAtLayer: layerId => {
+        const viewer = cesium.current?.cesiumElement;
+        if (!viewer || viewer.isDestroyed()) return;
+        const e = viewer.entities.getById(layerId);
+        if (!e) return;
+        viewer.flyTo(e);
```

Comment:
const camera = getCamera(viewer);
        viewer.zoomTo(
          e,
          new Cesium.HeadingPitchRange(camera?.heading ?? 0, camera?.pitch ?? -90, 5000),
        );

Is this a better solution?